test/specs/lightbox.js

```diff
@@ -1,11 +1,66 @@
-/* eslint-env jasmine */
-
+import $ from 'jquery';
 import lightbox from '../../lightbox';
 
 describe('lightbox', function() {
-  describe('init', function() {
-    it('should be a function', function() {
-      expect(lightbox.init).toEqual(jasmine.any(Function));
+  const FADE_TIME = 200;
+  const BLOCKING_CLASS = '.js-blocking';
+  const LIGHTBOX_CLASS = '.js-lightbox';
+  const NEXT_CLASS = '.js-next';
+  const CLOSE_CLASS = '.js-close';
```

Comment:
Just a personal preference. I use `UPPER_SNAKE` for all constant values (strings, numbers) that are used throughout the entire module. I've seen the `let` convention for non-constants (which is what I prefer, btw) and I've seen the `this.hello = "world"` convention (since a test run  keeps the same `this` state in beforeEach, it, afterEach). I'll change it to a different convention if you believe that's the right thing to do.